ompi/mca/osc/base/osc_base_init.c

```diff
@@ -54,6 +54,11 @@ ompi_osc_base_select(ompi_win_t *win,
         ompi_osc_base_component_t *component = (ompi_osc_base_component_t*)
             ((mca_base_component_list_item_t*) item)->cli_component;
 
+        ret = component->osc_version.mca_register_component_params();
+        if (ret != OPAL_SUCCESS) {
+            continue;
+        }
```

Comment:
I don't think it's the right place - it's generic (not ucx) code, and this is per-window and not global initialization code

Reply:
BTW, seems it's called here:
https://github.com/open-mpi/ompi/blob/6bf2a093ff25b91802e7aa86a7d537ec33e72da9/ompi/mca/osc/ucx/osc_ucx_component.c#L183